doc/en/components/nextjs-usage.md

```diff
@@ -0,0 +1,255 @@
+      IgrButtonModule.register();
+      return IgrButton;
+    }
+  );
+  
+  return (
+      <IgButton variant="contained">
+        <span key="title">Click me</span>
+      </IgButton>
+  );
+}
+```
+
+However, if a more complex component is used like the [**IgrGrid**](grids/data-grid.md), which typically contains child components, it is essential not to dynamically import each and every child component. The component should be used like this:
+
+```tsx
+"use client";
+import dynamic from "next/dynamic";
+import CustomersDataLocal from "./CustomersDataLocal.json";
+import "igniteui-react-grids/grids/combined";
+import "igniteui-react-grids/grids/themes/light/bootstrap.css";
+
+export default function GridDynamicComponent() {
+  const IgnGrid = dynamic(
+    async () => {
+      const {
+        IgrGrid,
+        IgrGridModule,
+        IgrColumn,
+        IgrGridToolbar,
+        IgrGridToolbarTitle,
+        IgrGridToolbarActions,
+        IgrGridToolbarPinning,
+      } = await import("igniteui-react-grids");
+      IgrGridModule.register();
+
+      const IgGrid = ({ ...props }) => {
+        return (
+          <IgrGrid {...props}>
+            <IgrGridToolbar>
+              <IgrGridToolbarTitle></IgrGridToolbarTitle>
+              <IgrGridToolbarActions>
+                <IgrGridToolbarPinning></IgrGridToolbarPinning>
+              </IgrGridToolbarActions>
+            </IgrGridToolbar>
+            <IgrColumn field="ID" header="ID" hidden="true"></IgrColumn>
+            <IgrColumn
+              field="CompanyName"
+              header="Company Name"
+              width="300px"
+            ></IgrColumn>
+            <IgrColumn
+              field="ContactName"
+              header="Contact Name"
+              width="200px"
+              pinned="true"
+            ></IgrColumn>
+            <IgrColumn
+              field="ContactTitle"
+              header="Contact Title"
+              width="200px"
+              pinned="true"
+            ></IgrColumn>
+            <IgrColumn
+              field="Address"
+              header="Address"
+              width="300px"
+            ></IgrColumn>
+            <IgrColumn field="City" header="City" width="120px"></IgrColumn>
+            <IgrColumn field="Region" header="Region" width="120px"></IgrColumn>
+            <IgrColumn
+              field="PostalCode"
+              header="Postal Code"
+              width="150px"
+            ></IgrColumn>
+            <IgrColumn field="Phone" header="Phone" width="150px"></IgrColumn>
+            <IgrColumn field="Fax" header="Fax" width="150px"></IgrColumn>
+          </IgrGrid>
+        );
+      };
+      return IgGrid;
+    }
+  );
+
+  return <IgnGrid data={CustomersDataLocal}></IgnGrid>;
+}
+```
+
+> [!Note]
+> Implementing lazy loading for components can enhance performance, but it is advisable to use it exclusively when the components are not immediately visible on the page.
```

Comment:
You should add a see also paragraph at the end with a few generic links to our docs and a link to nextjs docs too